clayui.com/content/docs/components/css-checkbox-radio.md

```diff
@@ -24,15 +20,15 @@ lexiconDefinition: 'https://liferay.design/lexicon/core-components/forms/radio-c
 	Don't forget to check <a href="https://www.w3.org/TR/wai-aria-practices/#checkbox">WAI-ARIA</a> accessibility pratices for checkboxes when writting your markup.
 </div>
 
-## Checkboxes and Radios
-
 Default checkboxes and radios are improved upon with the help of `.form-check`, **a single class for both input types that improves the layout and behavior of their HTML elements**. Checkboxes are for selecting one or several options in a list, while radios are for selecting one option from many.
 By [Bootstrap](https://getbootstrap.com/docs/4.1/components/forms/#checkboxes-and-radios)
 
-### Default (stacked)
+# Default
```

Comment:
We shouldn't have multiple `<h1>` tags on a page. I think the main categories should be an `h2` via `## some heading` and then nested items would be an h3